test/runTest.ts

```diff
@@ -3,11 +3,28 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as cp from 'child_process';
 import * as path from 'path';
-import { runTests } from 'vscode-test';
+import { downloadAndUnzipVSCode, resolveCliArgsFromVSCodeExecutablePath, runTests } from '@vscode/test-electron';
 
 async function main(): Promise<void> {
     try {
+        const vscodeExecutablePath = await downloadAndUnzipVSCode('1.66.0');
+        const [cli, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath);
+
+        cp.spawnSync(
+            cli,
+            [
+                ...args,
+                '--install-extension', 'ms-vscode.azure-account',
+                '--install-extension', 'ms-azuretools.vscode-azureresourcegroups',
+                '--install-extension', 'ms-python.python',
```

Comment:
FYI the gulp tasks for installing extensions we use do not actually install the extensions. They just place a fake package.json to fool VS Code into thinking the extensions are installed.

Reply:
Eventually we're going to have to just get rid of Gulp.